arch/generic/crc32_chorba_c.c

```
@@ -1062,9 +1070,11 @@ Z_INTERNAL uint32_t crc32_chorba_small_nondestructive(uint32_t crc, const uint64
     return crc32_braid(~crc, (uint8_t*)final, len-i);
 }
 
-#else // OPTIMAL_CMP == 64
+#else // CHORBA_W == 8
 
-Z_INTERNAL uint32_t crc32_chorba_small_nondestructive_32bit(uint32_t crc, const uint32_t *input, size_t len) {
+Z_INTERNAL uint32_t crc32_chorba_small_nondestructive_32bit(uint32_t crc, const uint8_t *input, size_t len) {
```

Cppcheck (2.19.0)

[style] The function 'crc32_chorba_small_nondestructive_32bit' should have static linkage since it is not used outside of its translation unit.

(staticFunction)